test/helpers/cilium.go

```diff
@@ -498,6 +498,7 @@ func (s *SSHMeta) PolicyWait(revisionNum int) *CmdRes {
 // ReportFailed gathers relevant Cilium runtime data and logs for debugging
 // purposes.
 func (s *SSHMeta) ReportFailed(commands ...string) {
+	fmt.Printf("===================== TEST FAILED =====================\n")
```

Comment:
Why not send that to `GinkgoWritter` or `wr`. Stdout output can be out of order in Ginkgo. I will suggest send to `wr`

Reply:
Done.